ironmon_tracker/screens/ViewLogWarningScreen.lua

```diff
@@ -0,0 +1,149 @@
+ViewLogWarningScreen = {
+	Labels = {
+		-- First warnings paragraph. Second value is text color
+		warnings1 = {
+			{ "Are you sure you want to view the",   "Intermediate text" },
+			{ "log file?",                           "Intermediate text" },
+			{ "",                                    "" },
+			{ "In Ironmon, it's against the spirit", "" },
+			{ "of the challenge to view log info",   "" },
+			{ "about the game before it's over.",    "" },
+		},
+		-- Second warnings paragraph, displayed below "yes" button
+		warnings2 = {
+			{ "If you are unsure, simply do not", "" },
+			{ "view the log file.",               "" }
+		},
+		header = "! ! W A R N I N G ! !",
+		yes = "Yes, I'm sure",
+	},
+	Colors = {
+		text = "Lower box text",
+		border = "Lower box border",
+		boxFill = "Lower box background",
+	},
+}
+
+local buttonHeight = 16
+local buttonWidth = 75
+-- Used to more easily place the "Yes" button
+local buttonYOffset = Constants.SCREEN.MARGIN + 10 +
+	 ((#ViewLogWarningScreen.Labels.warnings1 + 1) * (Constants.SCREEN.LINESPACING))
+-- Center the "Yes" button
+local buttonXOffset = Constants.SCREEN.WIDTH + Constants.SCREEN.MARGIN +
+	 ((Constants.SCREEN.RIGHT_GAP - (Constants.SCREEN.MARGIN * 2)) / 2) - (buttonWidth / 2)
+
```

Comment:
Wasn't sure of a better place to put these variables. Made these to avoid manual re-calculation of pixel co-ordinates and also to avoid hard-coding values